CONTRIBUTING.md

```diff
@@ -1 +1,91 @@
 # Contributing
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
+   build.
+2. Update the CHANGELOG.md with details of changes.
+3. Increase the version numbers in the package.json to the new version that this
```

Comment:
I dont think we want people changing our version. We should be handling that during our release process in my opinion.